src/EnrollU2FController.js

```diff
@@ -63,7 +65,7 @@ function (Okta, FormType, FormController, Footer, Q) {
           u2f.register(appId, registerRequests, [], function (data) {
             self.trigger('errors:clear');
             if (data.errorCode && data.errorCode !== 0) {
-              deferred.reject({ responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}});
+              deferred.reject({xhr: {responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}}});
```

Comment:
This change shows the valid error message instead of throwing TypeError when trying to read `responseJSON` from `undefined` inside the code.